EnglishFilter/sections/specific.txt

```diff
@@ -276,7 +276,7 @@ yoshare.net##iframe[id^="yoshare.net_"][id$="_DFP"]
 animax.*##[src*="//a.exdynsrv.com/iframe.php"]
 ||animax.*/ax.js
 dekki.com##div[pb-serve-label*="advert"]
-cuts-url.com##.content
+earncoin.site,cuts-url.com##.content
```

Comment:
It shouldn't be blocked. Not an ad.

Reply:
It's crap though, would you read it while waiting for the link?

Reply:
It's their description. Yes, it is located aggressively.
So, the only option is to add the rules to the Annoyances filter -> `annoyances.txt` with task line comment.
Something like that:
```
earncoin.site##.box-main > div.faq
earncoin.site##.short div.row > div.text-center > div.content
```

Reply:
Ok added.

Reply:
@wout-codes 
I've added them